thorlcr/graph/thgraphslave.cpp

```diff
@@ -1028,7 +1028,7 @@ class CThorCodeContextSlave : public CThorCodeContextBase, implements IEngineCon
     virtual bool allowDaliAccess() const
     {
         // NB. includes access to foreign Dalis.
-        return globals->getPropBool("Debug/@slaveDaliClient") || job.getWorkUnitValueBool("slaveDaliClient", false);
```

Comment:
There is at least one other instance of code that calls getWorkUnitValueBool("slaveDaliClient") that should also be changed?